docs/use_cases/recommender_systems.md

```diff
@@ -1,15 +1,111 @@
-<!-- TODO: Replace this text with a summary of article for SEO -->
+<!--SEO SUMMARY: Recommender Systems can be improved by incorporating side information about users and items, even when this information is scarce. According to an IBM study, personalized product recommendations can lead to a 10-30% increase in revenue. However, creating truly personalized experiences requires extensive user and product data, which is not always available. For example, providers with restrictions on collecting user data may only have basic product information like genre and popularity. While collaborative filtering approaches can still work reasonably well, they leave significant gains on the table. By incorporating side data, no matter how limited, recommendation quality can potentially increase significantly (over 20% in precision), leading to greater customers's satisfaction, more traffic, millions in additional revenue. -->
```

Comment:
I am not sure I understand point 1 - how would you write it?

Reply:
@levrone1987 Clarifying point 1: 
I think because it wasn't clear below whether you were comparing collaborative filtering vs. incorporating side data. I'm surmising that you're saying "collaborative filtering without side data incorporation leaves potential gains on the table." 
If that's accurate, then make it clear that you are talking about incorporating side data into a recommendation system that uses collaborative filtering, however you want to word it.

Reply:
Ok, thanks, @robertdhayanturner